pkg/healthcheck/healthcheck_output.go

```diff
@@ -210,16 +217,16 @@ func runChecksTable(wout io.Writer, hc Runner, output string) bool {
 		printResultDescription(wout, status, result)
 	}
 
+	var headerPrinted bool
 	prettyPrintResultsExtensionShort := func(result *CheckResult) {
-		lastCategory = printCategory(wout, lastCategory, result)
-
 		// bail out early and skip printing if we've got an okStatus
-		// Note: we still print the category headers to show which
-		// extension check we are running.
 		if result.Err == nil {
 			return
 		}
 
+		headerPrinted = printHeader(wout, headerPrinted, hc)
```

Comment:
After removing `ExtensionShortOutput`, the `headerPrinted` checks would move up to `prettyPrintResultsShort`.